HUD/SetMinimapComponent.md

```diff
@@ -5,24 +5,38 @@ ns: HUD
 
 ```c
 // 0x75A9A10948D1DEA6 0x419DCDC4
-Any SET_MINIMAP_COMPONENT(int componentID, BOOL toggle, int hudColor);
+int SET_MINIMAP_COMPONENT(int componentID, BOOL toggle, int hudColor);
 ```
 
-This native is used to colorize certain map components like the army base at the top of the map.
+This native is used to colorize/toggle certain map components like the army base.
 
-An incomplete list of components ID:
+An incomplete list of component id's:
 
-0: Los Santos' air port yellow lift-off markers.
-1: Sandy Shore's air port yellow lift-off markers.
-2: Trevor's air port yellow lift-off markers.
-6: Vespucci Beach lifeguard building.
-15: Army base.
+* **0** Los Santos International Airport yellow runway markers
+* **1** Sandy Shores Airfield yellow runway markers
+* **2** McKenzie Field yellow runway markers
+* **6** Vespucci Beach lifeguard building
+* **7** Top level zone of Alien Camp (Hippy Camp)
+* **8** Paleto Bay fire station drill tower
+* **10** Pala Springs Aerial Tramway
+* **11** Galileo Observatory power unit
+* **12** Small zone (empty "interior") near Central Los Santos Medical Center
+* **13** Richman Mansion grotto
+* **14** 2 Alien Camp (Hippy Camp) circles
+* **15** Fort Zancudo
 
 [List of hud colors](https://docs.fivem.net/docs/game-references/hud-colors/)
 
 ## Parameters
-* **componentID**: The component ID to change.
-* **toggle**: True to enable the color, false to disable the effect.
+* **componentID**: The component id to change.
+* **toggle**: True to enable/colorize, false to disable.
 * **hudColor**: The hudcolor index.
 
 ## Return value
+Seems to always return 1 (int) regardless of input
```

Comment:
It indeed always returns `1`, it's hard coded to that in the native.

Reply:
Thanks for clarifying that, I have clarified this in a new commit.